incognia.go

```diff
@@ -239,6 +257,10 @@ func (c *Client) RegisterPayment(payment *Payment) (ret *TransactionAssessment,
 }
 
 func (c *Client) registerPayment(payment *Payment) (ret *TransactionAssessment, err error) {
+	if payment == nil {
+		return nil, ErrMissingPayment
+	}
```

Comment:
Should we return a specific error for this case or consider that when the encapsulating object is missing it is actually the same as one of its fields being empty (i.e. missing installation ID)?